packages/story-editor/src/components/checklist/checks/pageBackgroundLowTextContrast/check.js

```diff
@@ -49,16 +47,6 @@ import { getSpansFromContent } from '../../utils';
  * @property {number} b blue value
  */
 
-/**
- *
- * @param {number} fontSize The text element's font size in editor pixels
- * @return {number} the true font size for measuring against accessibility standards
- */
-function getPtFromEditorFontSize(fontSize) {
-  // 1 point = 1.333333 px
-  return dataFontEm(dataToFontSizeY(fontSize, 100)) * 1.333333;
-}
```

Comment:
22pt in the editor would become `6.3999984px`
<img width="1146" alt="Screen Shot 2021-11-18 at 11 07 22 AM" src="https://user-images.githubusercontent.com/22185279/142452286-c4da35fa-b28a-44bb-b625-4eb88c2cca70.png">

Seems like we probably shouldn't remove this entirely - but rather we should make sure this translates to the exact calculated px value in the editor
.

Reply:
@miina @swissspidy Wanted to check with you two specifically, I'm hoping you have context here.

Should we be passing in the value that users have set in the design panel or the calculated value on the element? So in the above example, it would be `22px` vs `19.22px` 🤔

Reply:
@samwhale We should be using the attribute value of the element directly (`element.fontSize`), that would be 22 in this case. The "real" displayed value at that point (19.22px) depends completely on the screen size, so if we'd use that, it would create inconsistent results depending on the screen size.

Reply:
Awesome, so this should stay removed all together. Thanks!